src/groovy/au/org/emii/portal/wms/GaServer.groovy

```diff
@@ -0,0 +1,24 @@
+package au.org.emii.portal.wms
```

Comment:
As discussed, this isn't really GA specific - rather its WCS layer specific - worth considering how we handle this when we come to look at removing the filter extension as this filter would likely be common to all WCS layers and optionally time/elevation.

Reply:
In the first case I guess just specifying what it is for each layer would be OK as that's what we do for all our other layers.